core/src/main/java/org/keycloak/crypto/SignatureContext.java

```diff
@@ -0,0 +1,11 @@
+package org.keycloak.crypto;
+
+public interface SignatureContext {
```

Comment:
Maybe `SignatureSignerContext` would be more in line with complementary `SignatureVerifierContext` (cf. SignatureProvider)?

Reply:
Done